pg_search/tests/pg_regress/sql/json_groupby_orderby_limit.sql

```diff
@@ -0,0 +1,72 @@
+-- Test JSON field GROUP BY with aggregate custom scan
```

Comment:
Please add tests for edge cases:
   - Very large LIMIT values
   - LIMIT 0 or LIMIT 1
   - ORDER BY with NULL values
   - Multi-column ORDER BY scenarios
   - ORDER BY on columns with different collations/locales (and compare with PG results)